frontend/src/components/G6Graph.svelte

```diff
@@ -67,196 +84,194 @@
     const node: Item = graph.findById(event.id);
     if (node instanceof Node) {
       const edges = node.getEdges();
-
       for(const edge of edges){
         edge.toBack();
       }
     }
     graph.paint();
   };
-
-  export const updateTimeBar = (timeBarEnabled: boolean) => {
-    graph.removePlugin(graph.get("plugins")[1]); // changed index to 1 since the timebar is added after the tooltip
-    if (!timeBarEnabled) {
-      //TO-DO Reset the graph if wanted later
-    } else {
-      graph!.addPlugin(
-        new G6.TimeBar({
-          className: "g6TimeBar",
-          x: 0,
-          y: 0,
-          width: 500,
-          height: 110,
-          padding: 10,
-          type: "trend",
-          changeData: false,
-          trend: {
-            data: timeBarData,
-            smooth: true,
-          },
-          tick: {
-            tickLabelFormatter: (timeBarData: any) => {
-              return "";
-            },
-
-            tickLineStyle: {
-              fill: "#f28c18",
```

Comment:
Not what the reason for moving so much code around was 😅 It makes it really difficult to find the code written for this feature.